src/app/core/app.component.html

```diff
@@ -47,16 +45,10 @@
 <p-toast></p-toast>
 <p-confirmDialog></p-confirmDialog>
 <p-confirmPopup key="commonPopup"></p-confirmPopup>
-<p-dialog
-  *ngrxLet="fatalError$"
-  i18n-header="@@auth-error-modal-title" header="Error"
-  [visible]="true"
-  [style]="{width: '50vw'}"
-  [modal]="true"
-  [closeOnEscape]="false"
-  [closable]="false"
->
-  <p i18n="@@auth-error-modal-context">Oops, we are unable to make the site work properly. Are you connected to the Internet?</p>
+<p-dialog *ngrxLet="fatalError$" i18n-header="@@auth-error-modal-title" header="Error" [visible]="true"
+  [style]="{width: '50vw'}" [modal]="true" [closeOnEscape]="false" [closable]="false">
+  <p i18n="@@auth-error-modal-context">Oops, we are unable to make the site work properly. Are you connected to the
+    Internet?</p>
```

Comment:
Why is this change? The previous version follows the way we display multiline html.

Reply:
Sorry, default HTML formatter had modified this and I hadn't noticed it.